### ISSUE TITLE
Add support for AsyncAPI 2.1.0

### ISSUE THREAD

**Issue by galvo** [asyncapi/playground#147]
#### Reason/Context
Please try answering few of those questions
- Why we need this improvement? 
The latest AsyncAPI is now 2.1.0 but playground only supports 2..0.0 via the latest docker image https://hub.docker.com/r/ryts/asyncapi-playground/tags?page=1&ordering=last_updated, however https://playground.asyncapi.io/ supports 2.1.0, is it possible to release the update with  2.1.0 support?

- How will this change help?
Inline with latest AsyncAPI
- What is the motivation? 

#### Description
Please try answering few of those questions
- What changes have to be introduced?
- Will this be a breaking change?
- How could it be implemented/designed?

**Comment by derberg** [comment]
@galvo hey, the problem is that the image you are referring to is not official AsyncAPI image maintained by us. Playground doesn't have an official image published to docker or any other registry. There is a Dockerfile but just for users to build on their own.

**Comment by galvo** [comment]
Ok cool, can we get a official release of playground on Github that way i can tag and push a docker image to align with it

**Comment by derberg** [comment]
we were always hesitant to do it because of the works we plan for new version of the playground, that will also have new name. but on the other hand why not publishing it now and then just deprecate the image and point to the new one 🤔 

@fmvilas what do you think?
@galvo would you be able to help? we have some ready workflow for docker and I think it is enough just to copy-paste but for even so trivial tasks we need support,

**Comment by fmvilas** [comment]
Yeah, I don't think we'll have Studio ready to replace Playground any time soon so let's go ahead and publish a Docker image 👍

**Comment by galvo** [comment]
Happy to help out where i can

**Comment by derberg** [comment]
@galvo awesome! we basically need a workflow like https://github.com/asyncapi/generator/blob/master/.github/workflows/release-docker.yml and related npm scripts in place. As for the secrets, I would just have to make sure they are globally available in entire organization as for not they are only in `generator`

**Comment by galvo** [comment]
ok leave that with me

**Comment by magicmatatjahu** [comment]
@galvo We will be archiving the Playground in the near future so I suggest you to use the https://studio.asyncapi.com/ which is the successor of the Playground but with that same and much more capabilities. We also have a Docker Image - https://github.com/asyncapi/studio#using-it-via-docker. We are no longer mainataning the playground, we will be redirecting to the studio. Sorry for problems!
